core/src/main/java/org/kiwix/kiwixmobile/core/dao/entities/BookmarkEntity.kt

```diff
@@ -44,8 +44,8 @@ data class BookmarkEntity(
 
   private constructor(bookmark: Bookmark, zimFilePath: String?, favicon: String?) : this(
     0,
-    bookmark.zimId,
-    bookmark.zimName,
+    bookmark.zimId!!,
+    bookmark.zimName!!,
```

Comment:
We don't encourage to use double bang operator.

Reply:
done